packages/template-retail-react-app/app/components/promo-popover/index.jsx

```diff
@@ -56,25 +56,25 @@ const PromoPopover = ({header, children, ...props}) => {
                         })}
                     />
                 </PopoverTrigger>
-                <Portal>
-                    <PopoverContent border="none" borderRadius="base">
-                        <Box boxShadow="lg" zIndex="-1">
-                            <PopoverArrow />
-                            <PopoverCloseButton />
-                            <PopoverHeader borderBottom="none">
-                                {header || (
-                                    <Text fontWeight="bold" fontSize="md">
-                                        <FormattedMessage
-                                            defaultMessage="Promotions Applied"
-                                            id="promo_popover.heading.promo_applied"
-                                        />
-                                    </Text>
-                                )}
-                            </PopoverHeader>
-                            <PopoverBody pt={0}>{children}</PopoverBody>
-                        </Box>
-                    </PopoverContent>
-                </Portal>
+                <PopoverContent border="none" borderRadius="base">
+                    <Box boxShadow="lg">
+                        <PopoverArrow />
+                        <PopoverCloseButton />
+                        <PopoverHeader borderBottom="none" tabindex="0">
+                            {header || (
+                                <Text fontWeight="bold" fontSize="md">
+                                    <FormattedMessage
+                                        defaultMessage="Promotions Applied"
+                                        id="promo_popover.heading.promo_applied"
+                                    />
+                                </Text>
+                            )}
+                        </PopoverHeader>
+                        <PopoverBody pt={0} tabindex="0">
```

Comment:
```suggestion
                        <PopoverBody pt={0}>
```
The popover body is not an interactive element, so it should not have a tab index. VoiceOver reads the tooltip content, even without the tabindex:

![Screenshot of VoiceOver reading "Close, button, Promotions Applied description, Free Shipping Amount Above 50, tooltip"](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/62956339/e3e32e24-6593-4f6a-bcc3-00a5b5dd8b7c)

Reply:
@wjhsf How did you get it to read the tooltip content? I could not. It only read "Close, button"

Reply:
🤷 I just used VoiceOver, didn't do anything special.

Reply:
Had the same behavior as @wjhsf where the content of the tooltip is read out when I focus on the close button

Reply:
It seem VoiceOver is working for those tooltip. I've removed the tabIndex.